Rakefile

```diff
@@ -7,7 +7,7 @@ require_relative './ext/tiny_tds/extconsts'
 
 SPEC = Gem::Specification.load(File.expand_path('../tiny_tds.gemspec', __FILE__))
 
-ruby_cc_ucrt_versions = "3.3.0:3.2.0:3.1.0".freeze
+ruby_cc_ucrt_versions = "3.4.0:3.3.5:3.2.0:3.1.0".freeze
```

Comment:
Since 3.3.0 became 3.3.5, do we also want to update to 3.1.6 & 3.2.6?

Reply:
The change to 3.3.5 comes from the v1.6.0 release notes of `rake-compiler-dock`:
https://github.com/rake-compiler/rake-compiler-dock/releases/tag/v1.6.0

looking at how `nokogiri` configured their cross-compilation list, it appears to not be necessary to update any other version:
https://github.com/sparklemotion/nokogiri/blob/dadec3409556be85aecf9eff3857b883cacbe563/.cross_rubies